tasks/release.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+tasks:
+  snapshot:
+    desc: Creates a snapshot of the current working directory with files in .drainpipeignore excluded. The target directory should be provided as a cli argument e.g. task release:snapshot -- /tmp/release
+    cmds:
+      - if [ ! -f ".drainpipeignore" ]; then echo ".drainpipeignore does not exist" && exit 1; fi
+      - if [ -d "{{.CLI_ARGS}}" ] || [ "" == "{{.CLI_ARGS}}" ]; then echo "Please provide a path to a directory that does not yet exist" && exit 1; fi
+      - cp -R . {{.CLI_ARGS}}
+      - mv {{.CLI_ARGS}}/.drainpipeignore {{.CLI_ARGS}}/.gitignore
+      - cd {{.CLI_ARGS}} && git add . && git clean -ffdX
```

Comment:
How would I ignore a file that's committed to git, but shouldn't be in build artifacts? For example, developer documentation, or tests?

```sh
ls -la
total 24
drwxr-xr-x   6 andrew  wheel  192 25 Jun 16:17 .
drwxrwxrwt  22 root    wheel  704 25 Jun 16:16 ..
drwxr-xr-x  14 andrew  wheel  448 25 Jun 16:18 .git
-rw-r--r--   1 andrew  wheel    8 25 Jun 16:17 .gitignore
-rw-r--r--   1 andrew  wheel   25 25 Jun 16:17 exclude
-rw-r--r--   1 andrew  wheel   21 25 Jun 16:17 keep

$ git ls-tree HEAD
100644 blob b4317319340befd65497fd2b0494a19687f71e54	exclude
100644 blob ee366ce90075690eeeb1fc9c13416ced3f188916	keep

$ cat .gitignore
exclude

$ ls -la
total 24
drwxr-xr-x   6 andrew  wheel  192 25 Jun 16:17 .
drwxrwxrwt  22 root    wheel  704 25 Jun 16:16 ..
drwxr-xr-x  14 andrew  wheel  448 25 Jun 16:18 .git
-rw-r--r--   1 andrew  wheel    8 25 Jun 16:17 .gitignore
-rw-r--r--   1 andrew  wheel   25 25 Jun 16:17 exclude
-rw-r--r--   1 andrew  wheel   21 25 Jun 16:17 keep
```

Note that "exclude" is still there.

Reply:
That should work now